### PR TITLE
Install fastcomp without static libs

### PR DESCRIPTION
**Pull request by dschuff**
The trunk LLVM install uses INSTALL_TOOLCHAIN_ONLY to avoid getting all
the static libraries, and copies a few extra tools. Use the same logic
for fastcomp, which should cut down the tarball size.

### CODE REVIEW

File: src/build.py [hunk 1]
```diff
@@ -694,6 +693,23 @@ def Which(name):
     return ''
 
 
+def CopyLLVMTools(build_dir, install_subdir=''):
+  # The following isn't useful for now, and takes up space.
+  Remove(os.path.join(INSTALL_DIR, install_subdir, 'bin', 'clang-check'))
+  # The following are useful, LLVM_INSTALL_TOOLCHAIN_ONLY did away with them.
+  extra_bins = ['FileCheck', 'lli', 'llc', 'llvm-as', 'llvm-dis', 'llvm-link',
+                'llvm-nm', 'opt']
+  extra_libs = ['libLLVM*.so']
```
Comment (sbc100) [inline src/build.py]: I think you can remove this line now that I fixed it upstream to be installed with LLVM_INSTALL_TOOLCHAIN_ONLY

File: src/build.py [hunk 2]
```diff
@@ -694,6 +693,23 @@ def Which(name):
     return ''
 
 
+def CopyLLVMTools(build_dir, install_subdir=''):
```
Comment (sbc100) [inline src/build.py]: I would use 'prefix=' here and above (or at least be consistent).  'prefix' is the autoconf name for this kind of thing.

Comment (dschuff) [inline src/build.py]: Yeah, except it's not really a prefix, it's more of an infix (it is actually used as a subdir of the `INSTALL_DIR` which is what you might call the global prefix).

Comment (sbc100) [inline src/build.py]: In autoconf terminology this is called the PREFIX, and the its appended to the DESTDIR.   So you might configure with PREFIX=/usr/local and then install with DESTDIR=/tmp/location, and you'd end up with all your files in /tmp/location/usr/local.

**Pull request merged**